subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/DefaultDependencyManagementServices.java

```diff
@@ -245,7 +254,8 @@ DependencyHandler createDependencyHandler(Instantiator instantiator, Configurati
                 resolutionQueryFactory,
                 attributesSchema,
                 artifactTransformRegistrations,
-                artifactTypeRegistry);
+                artifactTypeRegistry,
+                instantiator);
```

Comment:
I couldn't figure out how to pass a `ServiceFactory` into this thing like we do into `Settings`, and `Project` but then I figured out that I didn't need to.